.stylelintrc.yml

```diff
@@ -0,0 +1,10 @@
+extends:
```

Comment:
Maybe worth of migrating to @oacore/configs, what do you think?

Reply:
Obviously I can handle that, you don't need to spending time on this only if you want to. 🙂

Reply:
See above